ReactTask/src/views/PatientProfile.js

```diff
@@ -0,0 +1,254 @@
+                <StylArrowVisitDoctor focusSelect={isFocusInputName} />
+                <StylInputVisitDoctor
+                  value={isValueInputName}
+                  onChange={(e) => setValueInputName(e.target.value)}
+                  onBlur={() => {
+                    setTimeout(() => {
+                      if (!nameDoctorsArr.includes(isValueInputName)) {
+                        setValueInputName('');
+                      }
+
+                      setFocusInputName(false);
+                    }, 100);
+                  }}
+                  onFocus={() => setFocusInputName(true)}
+                />
+                <StylBoxMenuSelectVisitDoctor focusSelect={isFocusInputName}>
+                  {nameDoctorsArr.map((item, index) => (
+                    <StylElementSelectVisitDoctor
+                      onClick={chooseNameDoctor}
+                      key={index}
+                    >
+                      {item}
+                    </StylElementSelectVisitDoctor>
+                  ))}
+                </StylBoxMenuSelectVisitDoctor>
+              </StylBoxColumnFlex>
+            </StylBoxColumnFlexMargin>
+            <StylBoxColumnFlexMargin dataReady={true}>
+              <StylTitleVisitDoctor>Note</StylTitleVisitDoctor>
+              <StylBoxColumnFlex>
+                <StylInputVisitDoctor placeholder='Leave a note if needed' />
+              </StylBoxColumnFlex>
+            </StylBoxColumnFlexMargin>
+          </StylItemChooseDoctor>
+          <StylItemChooseDoctor contentText='Choose a day for an appointment'>
+            <StylBoxColumnFlexMargin dataReady={isFilledData}>
+              <CalendarVisitDoctor changeData={changeDataChoose} />
+            </StylBoxColumnFlexMargin>
+          </StylItemChooseDoctor>
+          <StylItemChooseDoctor contentText='Select an available timeslot'>
+            <StylBoxColumnFlexMargin dataReady={isFilledData}>
+              <StylBoxChooseTime>
+                {possibleTimeArr.map((item, index) => (
+                  <StylBtnChooseTimeVisitDoctor
+                    key={index}
+                    type='button'
+                    isSelected={isTimeSelect === item}
+                    isDisabled={
+                      item === '2:00 pm' || item === '6:00 pm' ? true : false
+                    }
+                    onClick={(e) => setTimeSelect(e.currentTarget.innerHTML)}
+                  >
+                    {item}
+                  </StylBtnChooseTimeVisitDoctor>
+                ))}
+              </StylBoxChooseTime>
+            </StylBoxColumnFlexMargin>
+          </StylItemChooseDoctor>
+        </StylChooseDateAndTimeVisitDoctor>
+        <StylBtnSubmitFormVisitDoctor
+          isDisabled={isChooseAllData === false}
+          onClick={handleContent}
+        >
+          Submit
+        </StylBtnSubmitFormVisitDoctor>
+      </StylBoxPatientContent>
+    </StylBoxPatients>
+  );
+};
+
+export default PatientProfile;
```

Comment:
Katya or Maria at our lesson said that if component is bigger than 60 rows we should think how to split it

Reply:
Component `patientProfile` break apart

Reply:
Resolved